src/main/java/com/shopify/ShopifySdk.java

```diff
@@ -156,15 +161,30 @@
 	private String authorizationToken;
 	private WebTarget webTarget;
 	private String accessToken;
+	private Long minimumRequestRetryRandomDelayMilliseconds;
```

Comment:
Please change all of these Long classes to primitive longs. I cannot see why you need them to be classes.

minimumRequestRetryRandomDelayMilliseconds
maximumRequestRetryRandomDelayMilliseconds
maximumRequestRetryTimeoutMilliseconds

Probably even the connection and read timeouts.